views/js/test/runner/plugins/probes/latency/test.js

```diff
@@ -129,11 +129,12 @@ define([
             getProxy: function() {
                 return {
                     sendVariables: function(trace) {
-                        return new Promise(function() {
+                        return new Promise(function(resolve) {
                             var traceData = _.indexBy(probeData, function(entry) {
                                 return (entry.marker ? entry.marker + '-' : '') + entry.type + '-' + entry.id;
                             });
                             assert.deepEqual(trace, traceData, 'The trace data should have been provided');
+                            resolve();
```

Comment:
Since the promise is immediately resolving, no need to wrap here.
Can you simplify it by `Promise.resolve()`?

Reply:
sorry, but this resolve in the promise already and should to be run after some code, how do I need to replace it?

Reply:
I've extracted the code from the Promise